apps/vaporgui/VDoubleValidator.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <QDoubleValidator>
+
+//! \class VDoubleValidator
+//! Use as a direct replacement of QDoubleValidator.
```

Comment:
Please document how VDoubleValidator differs from QDoubleValidator

Reply:
done

Reply:
Is there a typo in here:

QDoubleValidator does not accept values outside
//! the range while VDoubleValidator does not allow the user to input
//! values outside of the range

It appears that they do the same thing

Reply:
QDoubleValidator allows users to input values outside the range.